frontend/src/api/network/notebooks.ts

```diff
@@ -87,8 +88,9 @@ const assembleNotebook = (data: StartNotebookData, username: string): NotebookKi
     tolerationSettings,
   } = data;
   const notebookId = overrideNotebookId || translateDisplayNameForK8s(notebookName);
+  const { notebookNamespace, dashboardNamespace } = useNamespaces();
   const resources: NotebookResources = { ...notebookSize.resources };
-  const imageUrl = `${image.imageStream?.status?.dockerImageRepository}:${image.imageVersion?.name}`;
+  const imageUrl = `${image.imageVersion?.from.name}`;
```

Comment:
```suggestion
  const imageUrl = image.imageVersion?.from.name || '';
```

Careful of `'undefined'` strings.

```
const obj = {};

`${obj.test}`
// 'undefined'

obj.test || ''
// ''
```

Reply:
As mentioned above, at runtime, this should have a from.name ...